ledger/acctupdates.go

```diff
@@ -90,17 +90,38 @@ func (au *accountUpdates) loadFromDisk(l ledgerForTracker) error {
 		return fmt.Errorf("accountUpdates.loadFromDisk: initAccounts not set")
 	}
 
+	latest := l.Latest()
 	err := au.dbs.wdb.Atomic(func(tx *sql.Tx) error {
 		var err0 error
-		err0 = accountsInit(tx, au.initAccounts, au.initProto)
-		if err0 != nil {
-			return err0
+		initAux := func() (basics.Round, error) {
+			err := accountsInit(tx, au.initAccounts, au.initProto)
+			if err != nil {
+				return 0, err
+			}
+
+			rnd, err := accountsRound(tx)
+			if err0 != nil {
+				return 0, err
+			}
+			return rnd, nil
 		}
 
-		au.dbRound, err0 = accountsRound(tx)
+		au.dbRound, err0 = initAux()
 		if err0 != nil {
 			return err0
 		}
+		// Check for blocks DB and tracker DB un-sync
+		if au.dbRound > latest {
+			// TODO: add assertions about non-archival to archival switch?
```

Comment:
I don't know about assertions, but clearly you need to log some info messages.

Reply:
Done